wire-ios/Wire-iOS/Sources/Developer/DebugAlert.swift

```diff
@@ -150,12 +157,18 @@ final class DebugLogSender: NSObject, MFMailComposeViewControllerDelegate {
         mailVC.setToRecipients([mail])
         mailVC.setSubject("iOS logs from \(userDescription)")
         mailVC.setMessageBody(message, isHTML: false)
-        mailVC.attachLogs()
+
         mailVC.mailComposeDelegate = alert
         alert.mailViewController = mailVC
 
         self.senderInstance = alert
-        controller.present(mailVC, animated: true, completion: nil)
+
+        Task {
+            await mailVC.attachLogs()
+            await MainActor.run {
+                controller.present(mailVC, animated: true, completion: nil)
+            }
```

Comment:
I think (not 100% sure) we can call without `MainActor.run` and to one line:
`await controller.present(mailVC, animated: true, completion: nil)`

Because `UIViewController` has the `@MainActor` attribute and thus all funcs should be automatically required to run on main thread. That's why it requires `await` prefix in this line.

Reply:
done